.github/workflows/build-test.yml

```diff
@@ -72,26 +72,15 @@ jobs:
           done
         shell: bash
 
-  install:
-    name: "Install"
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-
-      - name: Install node_modules
-        uses: OffchainLabs/actions/node-modules/install@main
```

Comment:
the latest install job checks for cache and runs restore if cache is not hit, so we don't have to split install & restore into two actions, and can use `install` at the current `restore` steps directly

https://github.com/OffchainLabs/actions/blob/main/node-modules/install/action.yml